src/hooks/useCalculatedColumns.ts

```diff
@@ -148,38 +151,19 @@ export function useCalculatedColumns<R, SR>({
     let left = 0;
     let totalFrozenColumnWidth = 0;
     let templateColumns = '';
-    let allocatedWidth = 0;
-    let unassignedColumnsCount = 0;
 
     for (const column of columns) {
-      let width = getSpecifiedWidth(column, columnWidths, viewportWidth);
-
-      if (width === undefined) {
-        unassignedColumnsCount++;
-      } else {
+      let width = columnWidths.get(column.key) ?? column.width;
+      if (typeof width === 'number') {
         width = clampColumnWidth(width, column);
```

Comment:
I wonder if we can replace this with setting the min/max widths on the header cells instead 🤔
Could min/max widths also accept strings?
[Could `clamp` replace min/max?](https://caniuse.com/css-math-functions)
Might not play well with virtualization?

Reply:
I guess `clamp` wouldn't work with manually resizing columns